docs/Contributing/visual-studio.rst

```diff
@@ -136,6 +136,37 @@ Studio will open a :strong:`CMake Overview Pages` tab.
 Microsoft provides documentation on CMake in Visual Studio --
 https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170
 
+Qt Visual Studio Tools
+======================
+
+There is an available extension called :strong:`Qt Visual Studio Tools`. This extension allows you to use the
+Qt Designer and other Qt tools inside of the Visual Studio IDE. This is especially useful if you want to edit
+any of the client :file:`.ui` files.
+
+Start by `downloading <https://www.qt.io/download-qt-installer>`_ the installer. Double-click the downloaded
+file and login with your Qt account. If you do not have a Qt account, there should be a link in the installer
+window to create one. Click :guilabel:`Next`, agree to the GNU License and state that you are an individual,
+not a business. Click :guilabel:`Next` and pick to either send or not send usage statistics to Qt.
+Click :guilabel:`Next`. Note that installing to :file:`C:\\Qt` is fine. Ensure :guilabel:`Custom Install`
+is selected and then click :guilabel:`Next`. On the right, uncheck :guilabel:`latest support release` and
+check :guilabel:`LTS`, click :guilabel:`Filter`. Expand :guilabel:`Qt` and check the ``5.15.2`` option. Click
+:guilabel:`Next` and agree to the Microsoft license. Click :guilabel:`Next` and note that :strong:`Qt` for
+the Start Menu is fine. Click :guilabel:`Next` and :guilabel:`Install` to begin the process.
+
+.. note:: You can reduce the size of the Qt Tools install by expanding the ``5.15.2`` option and unchecking
+  ``WebAssembly``, ``MSVC 2015 64-bit``, ``MSVC 2019 32-bit``, ``MinGW 8.1.0 32-bit``, ``MinGW 8.1.0 64-bit``,
+  ``UWP*``, and ``Android``. Unless you intend to develop for those platforms, you do not need to download and
+  install those components.
+
+Now to install the extension. In Visual Studio, click on
+:menuselection:`Extensions --> Manage Extensions --> Online`. In the upper right of the window, you will see
+a search box. Search for :strong:`Qt Visual Studio Tools`. Select it in the center and click the
+:guilabel:`Install` button. When it is finished downloading, close Visual Studio. A new window will
+come up that installs the extension. When finished re-open Visual Studio.
```

Comment:
Again, this seems a little verbose. I might be biased, but I'd expect a developer to know how to install extensions in their IDE.

Reply:
I shrunk this one down though.